backend/README.md

```
@@ -1,6 +1,87 @@
+      - `$ gcloud config set project PROJECT_ID`
+
+  - backend のコンテナ内で`sudo poetry install --no-root` ※google 関連パッケージを sudo 権限でインストール
+  - `python app/utils/gemini_request_stream.py`で gemini から出力されれば OK
+
+- パッケージを追加する場合
+  - Dev Container を起動してから`$ poetry add <パッケージ>`
+  - 開発環境のみのパッケージは`$ poetry add -D <パッケージ>`
+  - ローカル環境から操作したい場合
+    - バックエンド `$ docker compose exec backend poetry add <パッケージ>`
+
+### 設定ファイル
```

Markdownlint

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)